src/types/context.ts

```diff
@@ -1,7 +1,8 @@
-import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
 import { Octokit } from "@octokit/rest";
-import { PluginSettings } from "./plugin-inputs";
+import { EmitterWebhookEvent as WebhookEvent, EmitterWebhookEventName as WebhookEventName } from "@octokit/webhooks";
```

Comment:
Why do you rename?

Reply:
Did not rename, just reorganized import order because they were not sorted according to `prettier`. The diff is misleading.
<img width="1390" alt="image" src="https://github.com/ubiquibot/plugin-template/assets/9807008/7d8e9827-d8ff-4f5a-a47e-d354aa93a267">

Reply:
```suggestion
import { EmitterWebhookEvent, EmitterWebhookEventName } from "@octokit/webhooks";
```
So then it should be like this?

Reply:
It can be. Maybe that was to shorten the names. They have been renamed consistently on every plugin, and the kernel as well, in the same fashion.